dbfit-java/build.gradle

```diff
@@ -87,7 +87,7 @@ subprojects {
         classpath += sourceSets.main.runtimeClasspath
     }
 
-    ext.dbdeploy = { dbDriver, url, userid, password, delimiterType = 'normal' ->
+    ext.dbdeploy = { dbDriver, url, userid, password, delimiterType = 'normal', lineending = 'crlf' ->
```

Comment:
No. I couldn'get get `dbdeploy` to fire for Informix and couldn't work out why (I had to rebuild the VM for some reason) but gave up anyway as when in Informix mode it fails (no transaction support).